src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixAbstractMethodsStep.cs

```diff
@@ -297,6 +339,19 @@ void AddNewExceptionMethod (TypeDefinition type, MethodDefinition method)
 			}
 		}
 
+		bool markedAbstractMethodErrorType;
+
+		void MarkAbstractMethodErrorType () {
+			if (markedAbstractMethodErrorType)
+				return;
+			markedAbstractMethodErrorType = true;
+				
+			var td = AbstractMethodErrorConstructor.DeclaringType.Resolve ();
+			Annotations.Mark (td);
+			Annotations.SetPreserve (td, TypePreserve.Nothing);
+			Annotations.AddPreservedMethod (td, AbstractMethodErrorConstructor.Resolve ());
```

Comment:
Should use context.ResolveMethodDefinition () and check for null return value

Reply:
Seems the Resolve calls were unnecessary to begin with, so I just changed the type of AbstractwMethodErrorConstructor to MethodDefinition (same above).